proj/src/main/webapp/script.js

```diff
@@ -116,55 +116,49 @@ function appendMessage(payload){
     Realtime Database
  */
 
-const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
-const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
+var CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
+var tag = 'test';
+var dbRefObject = getDbRef(tag, CHAT_ID);
```

Comment:
Don't forget the `TODO:` to actually set the correct tag and move the getDbRef call to where it is actually needed.